solver/exporter.go

```diff
@@ -171,6 +171,12 @@ func (e *exporter) ExportTo(ctx context.Context, t CacheExporterTarget, opt Cach
 		for _, dep := range deps {
 			recs, err := dep.CacheKey.Exporter.ExportTo(ctx, t, opt)
 			if err != nil {
+				if opt.Mode == CacheExportModeMax {
```

Comment:
So this errors on `ModeMin`? What is that error?

Reply:
Also, to be clear, I have not hit any errors here in the real world. I only noticed this issue because I was reading through the code to get a better understanding of it.